contracts/VRTVault/VRTVaultStorage.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity ^0.5.16;
+import "../Utils/SafeMath.sol";
+import "../Utils/IBEP20.sol";
+
+contract VRTVaultAdminStorage {
+    /**
+    * @notice Administrator for this contract
+    */
+    address public admin;
+
+    /**
+    * @notice Pending administrator for this contract
+    */
+    address public pendingAdmin;
+
+    /**
+    * @notice Active brains of VRT Vault
+    */
+    address public vrtVaultImplementation;
```

Comment:
bscscan does not recognize `vrtVaultImplementation` as a pointer to the implementation contract. Let's use a conventional name here:

```suggestion
    address public implementation;
```

Reply:
renamed to implementation